apps/dotcom/client/src/tla/layouts/TlaSidebarLayout/TlaSidebarLayout.tsx

```diff
@@ -13,14 +14,16 @@ export function TlaSidebarLayout({ children }: { children: ReactNode; collapsabl
 		() => app.getSessionState().isSidebarOpenMobile,
 		[app]
 	)
+	const currentEditor = useValue('editor', () => globalEditor.get(), [])
 	usePreventAccidentalDrops()
+
 	return (
 		<div
 			className={styles.layout}
 			data-sidebar={isSidebarOpen}
 			data-sidebarmobile={isSidebarOpenMobile}
 		>
-			<TlaSidebar />
+			{currentEditor && <TlaSidebar />}
```

Comment:
when would we want to hide the sidebar if there's not an editor running?

Reply:
yeah, right now the sidebar is really entangled with the Sidebar which isn't great:
- `TlaSidebar` → `useTldrFileDrop` → `useToasts` → `TldrawUiDialogsProvider` → `editor`
- `TldrawUiContextProvider` → `useEditor`
- `TlaFileMenu` → `useDialogs` → `TldrawUiDialogsProvider` → `editor`

several things like that :-/